dataeng/jobs/analytics/SnowflakeRefreshSnowpipe.groovy

```diff
@@ -0,0 +1,66 @@
+package analytics
+
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_multiscm
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_parameters
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_wrappers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_publishers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_triggers
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm_parameters
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm
+
+
+class SnowflakeRefreshSnowpipe {
+    public static def job = { dslFactory, allVars ->
+        dslFactory.job('snowflake-refresh-snowpipe') {
+
+            // This job should be disabled until the rest of the work in DE-1779 is complete!
+            disabled(true)
+
+            logRotator common_log_rotator(allVars)
+            parameters secure_scm_parameters(allVars)
+            parameters {
+                stringParam('ANALYTICS_TOOLS_URL', allVars.get('ANALYTICS_TOOLS_URL'), 'URL for the analytics tools repo.')
+                stringParam('ANALYTICS_TOOLS_BRANCH', allVars.get('ANALYTICS_TOOLS_BRANCH'), 'Branch of analtyics tools repo to use.')
+                stringParam('PIPE_NAME', allVars.get('PIPE_NAME'), 'Name of Snowflake snowpipe to refresh')
+                stringParam('TABLE_NAME', allVars.get('TABLE_NAME'), 'Name of Snowflake table that receives snowpipe data')
```

Comment:
Is there a pattern to have these non-secret params just default to values here?

Reply:
I used the patter that we had in DX for iterating over a dsl with multiple non-secret values.